frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -13,25 +13,26 @@ import { parseCsvForError } from "../../utils/FileUtils";
 import { useWatersUploader } from "../../hooks/network/WatersHooks";
 import { useOrganizationSettings } from "../../hooks/UseOrganizationSettings";
 import { EventName, trackAppInsightEvent } from "../../utils/Analytics";
-import useSenderSchemaOptions from "../../senders/hooks/UseSenderSchemaOptions";
+import useSenderSchemaOptions, {
+    SchemaOption,
+} from "../../senders/hooks/UseSenderSchemaOptions";
 import { useSenderResource } from "../../hooks/UseSenderResource";
 import { RSSender } from "../../config/endpoints/settings";
 import { MembershipSettings } from "../../hooks/UseOktaMemberships";
+import site from "../../content/site.json";
+import { USExtLink } from "../USLink";
 
-import { FileHandlerForm } from "./FileHandlerForm";
-import {
-    RequestLevel,
-    FileQualityFilterDisplay,
-    FileSuccessDisplay,
-    RequestedChangesDisplay,
-} from "./FileHandlerMessaging";
+import { FileHandlerStepTwo } from "./FileHandlerStepTwo";
```

Comment:
It'd be better to name these steps something a little more meaningful about their functionality.  `One`, `Two`, `Three`, `Four` don't mean anything without knowing the order as-is right now, and it'd require cascading renames if we, say, added a step in between `Two` and `Three`.